### ISSUE TITLE
Blank item under network/wireless menu.

### ISSUE THREAD

**Issue by Darthpenguin** [julio641742/extend-panel-menu#73]
![extend panel issue](https://user-images.githubusercontent.com/3740366/39950520-a6b9796a-554f-11e8-8163-4013fc79614d.png)

Noticed this little problem with the extension on a fresh Arch Install. A blank menu item. Just an eye-sore really.

**Comment by benjbellon** [comment]
Noticed this error as well after downloading the extension. In my case, the `_settings` property on `NMDeviceWireless` does not exist. Specifically, the error occurs here:

https://github.com/julio641742/extend-panel-menu/blob/master/indicators/network.js#L141

The `_settings` property is needed to grab the AP connections:

https://github.com/julio641742/extend-panel-menu/blob/master/indicators/network.js#L289

The null case was previously unguarded (thus breaking on gnome shell 3.28). As above, the master branch has resolved these issues.

Presumably, you can just pull this file down until master is reflected in the gnome extensions "market."

**Comment by Darthpenguin** [comment]
Forgive me if this sounds dumb but I'm not completely following you. Are you saying there is a workaround for this? Would you be able to provide step-by-step on how to fix it? I would really appreciate it.

**Comment by Darthpenguin** [comment]
wait, I think I got it. I copied the content of the above linked file and overwrote .local/share/gnome-shell/extensions/extend-panel-menu@julio641742/indicators/network.js

Still not perfect though. It lists available WiFi networks with a checkbox next to the actively connected one. But then it also has a menu below that with the title of the connected SSID that offers me the options to select network, turn off, or go to wifi settings. Seems redundant to me. but it is better than a blank UI element.

**Comment by benjbellon** [comment]
@Darthpenguin yeah, I think that is a design choice (rather than a feature bug). That is also how it appears for me. Glad you managed to get it to work :+1:
